source/MRMesh/MRAABBTreeBase.h

```diff
@@ -61,3 +61,5 @@ class AABBTreeBase
 };
 
 } //namespace MR
+
+#include "MRAABBTreeBase.hpp"
```

Comment:
looks suspicious

Reply:
I will revert this, doesn't seem to be necessary.